cms/grading/__init__.py

```diff
@@ -448,7 +456,7 @@ def compilation_step(sandbox, commands):
 
 
 def evaluation_step(sandbox, commands,
-                    time_limit=0.0, memory_limit=0,
+                    time_limit=0.0, memory_limit=0, max_processes=1,
```

Comment:
So my suggestion for LANGUAGE_TO_MAX_PROCESSES['default'] was *because* that exists and looks like the "default" value to someone who's only examining the CMS code at 2am before a seln; i.e. it's the lease surprising value to have to change.

Also, I *believe* the three things that call evaluation_step are Batch.py, OutputOnly.Py and probably Codebreaker.py? Ideally, they all execute under the exact same environment for various reasons; in the case of CB hey, the more forks the better >_>